Sources/SwiftRefactor/ConvertComputedPropertyToZeroParameterFunction.swift

```diff
@@ -0,0 +1,133 @@
+      statements.trailingTrivia = statements.trailingTrivia.droppingTrailingWhitespace
+    case .getter(let codeBlock):
+      statements = codeBlock
+    }
+
+    let returnType = typeAnnotation.type
+
+    var returnClause: ReturnClauseSyntax?
+    let triviaAfterSignature: Trivia
+
+    if !returnType.isVoid {
+      triviaAfterSignature = .space
+      returnClause = ReturnClauseSyntax(
+        arrow: .arrowToken(
+          leadingTrivia: typeAnnotation.colon.leadingTrivia,
+          trailingTrivia: typeAnnotation.colon.trailingTrivia
+        ),
+        type: returnType
+      )
+    } else {
+      triviaAfterSignature = typeAnnotation.colon.leadingTrivia + typeAnnotation.colon.trailingTrivia
+    }
+
+    accessorBlock.leftBrace.leadingTrivia = accessorBlock.leftBrace.leadingTrivia.droppingLeadingWhitespace
+    accessorBlock.rightBrace.trailingTrivia = accessorBlock.rightBrace.trailingTrivia.droppingTrailingWhitespace
+
+    let body = CodeBlockSyntax(
+      leftBrace: accessorBlock.leftBrace,
+      statements: statements,
+      rightBrace: accessorBlock.rightBrace
+    )
+
+    var parameterClause = FunctionParameterClauseSyntax(parameters: [])
+    parameterClause.trailingTrivia = identifierPattern.identifier.trailingTrivia + triviaAfterSignature
+
+    let functionEffectSpecifiers = FunctionEffectSpecifiersSyntax(
+      leadingTrivia: effectSpecifiers?.leadingTrivia,
+      asyncSpecifier: effectSpecifiers?.asyncSpecifier,
+      throwsClause: effectSpecifiers?.throwsClause,
+      trailingTrivia: effectSpecifiers?.trailingTrivia.droppingTrailingWhitespace
+    )
+    let functionSignature = FunctionSignatureSyntax(
+      parameterClause: parameterClause,
+      effectSpecifiers: functionEffectSpecifiers,
+      returnClause: returnClause
+    )
+
+    return FunctionDeclSyntax(
+      modifiers: syntax.modifiers,
+      funcKeyword: .keyword(
+        .func,
+        leadingTrivia: syntax.bindingSpecifier.leadingTrivia,
+        trailingTrivia: syntax.bindingSpecifier.trailingTrivia
+      ),
+      name: identifierPattern.identifier.with(\.trailingTrivia, []),
+      signature: functionSignature,
+      body: body
+    )
+  }
+}
+
+fileprivate extension TypeSyntax {
+  var isVoid: Bool {
+    switch self.as(TypeSyntaxEnum.self) {
+    case .identifierType(let identifierType) where identifierType.name.text == "Void": return true
+    case .tupleType(let tupleType) where tupleType.elements.isEmpty: return true
+    default: return false
+    }
+  }
+}
+
+fileprivate extension Trivia {
+  var droppingLeadingWhitespace: Trivia {
+    return Trivia(pieces: self.drop(while: \.isWhitespace))
+  }
+
+  var droppingTrailingWhitespace: Trivia {
+    return Trivia(pieces: self.reversed().drop(while: \.isWhitespace).reversed())
+  }
+}
+
+fileprivate extension TokenSyntax {
+  var trivia: Trivia {
+    return leadingTrivia + trailingTrivia
+  }
+}
```

Comment:
Instead of adding these functions to multiple files, what do you think about adding a SyntaxUtils.swift file to `SwiftRefactor` and adding them there?

Reply:
Extracted all repetitive chunks to the new file, should we add a warning about inserting extensions in alphabetical order?

Reply:
I don’t expect that file to grow too much, so I don’t care that much about ordering and we shouldn’t need a warning about the alphabetical ordering.